src/utils/toast.tsx

```diff
@@ -4,6 +4,14 @@ import {toast} from 'react-toastify';
 import Toast from 'components/Toast';
 import {ToastType} from 'enums';
 
+/**
+ * Displays error toasts for each error message in the provided array.
+ * @param {string[]} errors - An array of error messages to be displayed as toasts.
+ */
+export const displayErrorToasts = (errors: string[]): void => {
+  errors.forEach(displayErrorToast);
+};
```

Comment:
The existing `displayErrorToast` would also have been used here but since I wanted to iterate through the errors list and I wasn't really willing to change the functionality/code of `displayErrorToast` so I created this another function